pkg/tests/testdata/expected/info_view-timescaledb-2.out

```diff
@@ -46,7 +46,7 @@ SELECT compress_chunk(show_chunks('prom_data.cpu_usage'));
 SELECT compress_chunk(show_chunks('prom_data.cpu_total'));
              compress_chunk             
 ----------------------------------------
- _timescaledb_internal._hyper_2_2_chunk
+ _timescaledb_internal._hyper_3_2_chunk
```

Comment:
I am not sure why is this change.

Reply:
because we are turning compression on earlier.